guides/doc-Provisioning_Guide/topics/proc_using-an-imagebuilder-image-for-provisioning.adoc

```diff
@@ -17,11 +17,11 @@ endif::[]
 . Use `scp` or `rsync` to copy the image using SSH to the base operating system of {Project}.
 ifeval::["{build}" == "satellite"]
 . On {Project}, create a custom product, add a custom file repository to this product, and upload the image to the repository.
-For more information, see {BaseURL}content_management_guide/managing_iso_images#importing_individual_iso_images_and_files[Importing Individual ISO Images and Files] in the _Content Management Guide_.
+For more information, see {ContentManagementDocURL}importing_individual_iso_images_and_files[Importing Individual ISO Images and Files] in the _Content Management Guide_.
 endif::[]
 ifeval::["{build}" == "foreman"]
 . If you use the Katello plug-in, on {Project}, create a custom product, add a custom file repository to this product, and upload the image to the repository.
-For more information, see {BaseURL}content_management_guide/managing_iso_images#importing_individual_iso_images_and_files[Importing Individual ISO Images and Files] in the _Content Management Guide_.
+For more information, see {ContentManagementDocURL}importing_individual_iso_images_and_files[Importing Individual ISO Images and Files] in the _Content Management Guide_.
```

Comment:
This does not work for Foreman:
https://docs.theforeman.org/master/Content_Management_Guide/index-foreman.html#importing_individual_iso_images_and_files

Reply:
For that, I have added an ID for importing_individual_iso_images_and_files in `guides/doc-Content_Management_Guide/topics/Managing_ISOs_and_Files.adoc`. After the PR is merged, this link will work.